cypress/support/verifyLinksUtil.ts

```diff
@@ -14,7 +14,9 @@ export function verifyLinks(path: string) {
             let errorMessage = response
               ? `Status code: ${response.status}`
               : 'No response received or request timed out'
-            cy.log(`Broken link: ${href} - ${errorMessage}`)
+            if (response && response.status !== 999) {
```

Comment:
What is a 999?

Reply:
Request Denied - we get it for the FF LinkedIn page

Reply:
Got it! I'm pushing a CR to abstract magic numbers like these and to refactor the code a bit for clarity and logic.

Reply:
Looks good, tho if it's ok with you I'd remove the logs of "Ignored links" - not the information we need? We just don't want to include them as broken links.

```
function isClientOrServerError(status: number) {
  return (
    status >= CLIENT_ERROR_THRESHOLD && status !== REQUEST_DENIED_STATUS_CODE
  )
}
```